### PR TITLE
Completed base requirements of React.js - Validated Input Component

### PR DESCRIPTION
**Pull request by ChrisGitmed**
react-validated-input-component

### CODE REVIEW

**Comment by ChrisGitmed** [comment]
Still finishing up the extra functionality.

File: react-validated-input-component/dist/styles.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.lnr-cross-circle {
+  color: red;
+  padding-left: 0.4rem;
+}
+
+.lnr-thumbs-up {
+  color: green;
+  padding-left: 0.4rem;
+}
```
Comment (thebearingedge) [inline react-validated-input-component/dist/styles.css:32]: 🤮 if these are defined in a library 🙅

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Overall great, except one no-no.

File: react-validated-input-component/src/validated-input.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+              flag = false;
+            }
+          }
+          if (flag) {
+            message = 'Your password must include a symbol.';
+          }
+        }
+        */
+      }
+    }
```
Comment (thebearingedge) [inline react-validated-input-component/src/validated-input.jsx:63]: I strongly recommend making separate methods to check each of these. They can return `null` or an error message string.

```js
isLongEnough() {

}

hasDigit() {

}

hasSpecialCharacter() {

}
```

Then you can call them in your render method and pick your icon based on the truthiness of the error message.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by ChrisGitmed** [comment]
Still need to complete!

File: react-validated-input-component/dist/styles.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  width: 100%;
+}
+
+.lnr-cross-circle {
+  color: red;
+}
+
+.lnr-thumbs-up {
+  color: green;
+}
```
Comment (thebearingedge) [inline react-validated-input-component/dist/styles.css:30]: We don't want to style 3rd-party classes.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice! one tiny tweak

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**